src/org/loklak/LoklakEmailHandler.java

```diff
@@ -0,0 +1,48 @@
+/**
+ *  LoklakEmailHandler
+ *  Copyright 25.05.2016 by Shiven Mian, @shivenmian
+ *
+ *  This library is free software; you can redistribute it and/or
+ *  modify it under the terms of the GNU Lesser General Public
+ *  License as published by the Free Software Foundation; either
+ *  version 2.1 of the License, or (at your option) any later version.
+ *  
+ *  This library is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ *  Lesser General Public License for more details.
+ *  
+ *  You should have received a copy of the GNU Lesser General Public License
+ *  along with this program in the file lgpl21.txt
+ *  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.loklak;
+
+import java.util.regex.Pattern;
+import java.util.regex.PatternSyntaxException;
+
+import org.eclipse.jetty.util.log.Log;
+
+public class LoklakEmailHandler {
+	
+	private String addressTo;
+	private String addressFrom;
+	private String subject;
+	private String text;
+	private Pattern pattern;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'pattern' could be replaced by a local variable.](https://www.codacy.com/app/sudheesh1995/loklak_server/file/2629824425/issues/source?bid=3288751&fileBranchId=3317765#l33)